packages/contract-wrappers/src/utils/constants.ts

```diff
@@ -14,4 +14,5 @@ export const constants = {
     ZERO_AMOUNT: new BigNumber(0),
     ONE_AMOUNT: new BigNumber(1),
     ETHER_TOKEN_DECIMALS: 18,
+    METAMASK_DENIED_SIGNATURE_PATTERN: 'MetaMask Tx Signature: User denied transaction signature',
```

Comment:
If you relax this a little you may also be able to catch Coinbase Wallet and some other wallet apps.

Looks like Coinbase Wallet does throw `User denied transaction signature` when rejecting a tx, might be worth relaxing to catch both?